docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -568,11 +568,13 @@ nested directories, or `false` to ignore them.
 (iceberg-add-files)=
 #### Add files
 
-The connector can add files from tables or locations if
+The connector can add files from tables or locations to an existing table if
```

Comment:
Also .. maybe we need to specify .. from Hive tables .. unless you can pull from other table formats as well